samples/bluetooth/direct_test_mode/sysbuild.cmake

```
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+get_property(PM_DOMAINS GLOBAL PROPERTY PM_DOMAINS)
+
+# Include app core image if enabled
+if(SB_CONFIG_SUPPORT_APPCORE AND DEFINED SB_CONFIG_APPCORE_IMAGE_NAME)
+  ExternalZephyrProject_Add(
+    APPLICATION ${SB_CONFIG_APPCORE_IMAGE_NAME}
+    SOURCE_DIR ${SB_CONFIG_APPCORE_IMAGE_PATH}
+    BOARD ${SB_CONFIG_APPCORE_REMOTE_BOARD_NAME}
+  )
+
+  if(NOT "${SB_CONFIG_APPCORE_IMAGE_DOMAIN}" IN_LIST PM_DOMAINS)
+    list(APPEND PM_DOMAINS ${SB_CONFIG_APPCORE_IMAGE_DOMAIN})
+  endif()
+
+  set_property(GLOBAL APPEND PROPERTY
+               PM_${SB_CONFIG_APPCORE_IMAGE_DOMAIN}_IMAGES
+               "${SB_CONFIG_APPCORE_IMAGE_NAME}"
+  )
+
+  set_property(GLOBAL PROPERTY DOMAIN_APP_${SB_CONFIG_APPCORE_IMAGE_DOMAIN}
```

Can't we just use CPUAPP instead of the SB_CONFIG_APPCORE_IMAGE_DOMAIN config?
```suggestion
  set_property(GLOBAL PROPERTY DOMAIN_APP_CPUAPP
```
Similarly, in other palces.
If so, the SB_CONFIG_APPCORE_IMAGE_DOMAIN config seems redundant.

---

You are correct. This config is redundant, removing it.